### PR TITLE
ref: Refactor alert rules API to return object instead of HTML

### PR DESCRIPTION
**Pull request by lynnagara**
Second attempt to refactor alert api

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/rules/actions/notify_event_service.py [hunk 1]
```diff
@@ -29,8 +29,16 @@ def __init__(self, *args, **kwargs):
 
 
 class NotifyEventServiceAction(EventAction):
-    form_cls = NotifyEventServiceForm
-    label = 'Send a notification via {service}'
+    def __init__(self, *args, **kwargs):
+        super(NotifyEventServiceAction, self).__init__(*args, **kwargs)
+        self.label = 'Send a notification via {service}'
+        self.form_cls = NotifyEventServiceForm
```
Comment (EvanPurkhiser) [inline src/sentry/rules/actions/notify_event_service.py]: Why move these two into the instance init?

**Review by evanpurkhiser — COMMENTED** [review]
Not sure if this is still a WIP, but it's looking good, love this.

File: src/sentry/rules/actions/notify_event_service.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/rules/actions/notify_event_service.py]: No particular reason, will move back

**Comment by lynnagara** [comment]
@EvanPurkhiser This is ready for review

**Comment by EvanPurkhiser** [comment]
You might want to take a look at the slack notify action as well. It's over under `integrations/slack`

**Pull request closed without merging**